src/main/java/io/logz/jmx2logzio/Jmx2LogzioJavaAgent.java

```diff
@@ -28,18 +30,31 @@
     private static final String FILE_SYSTEM_SPACE_LIMIT = "FILE_SYSTEM_SPACE_LIMIT";
     private static final String CLEAN_SENT_METRICS_INTERVAL = "CLEAN_SENT_METRICS_INTERVAL";
     private static final String EXTRA_DIMENSIONS = "EXTRA_DIMENSIONS";
-    private static final Logger logger = LoggerFactory.getLogger(Jmx2LogzioJavaAgent.class);
+    private static final String LOG_LEVEL = "LOG_LEVEL";
+    private static final Logger logger = (Logger) LoggerFactory.getLogger(Jmx2LogzioJavaAgent.class);
     private static final String JAVA_AGENT_CONFIGURATION_FILE = "javaagent.conf";
     private static final int SPLIT_KEY_VALUE_COUNT_LIMIT = 2;
     private static final int INDEX_OF_KEY = 0;
     private static final int INDEX_OF_VALUE = 1;
+    private static final String LOG_LEVEL_DEBUG = "DEBUG";
 
     public static void premain(String agentArgument, Instrumentation instrument) {
 
         logger.info("Loading with agentArgument: {}", agentArgument);
         Config finalConfig = getIntegratedConfiguration(agentArgument);
-        Jmx2LogzioConfiguration jmx2LogzioConfiguration = new Jmx2LogzioConfiguration(finalConfig);
+        Level logLevel = Level.WARN;
```

Comment:
If in the xml it's already WARN why do you need to set warn as a default here?

Reply:
the xml property refers to java.utils.logging.Logger and not to logback's logger